config/routes.rb

```diff
@@ -6,7 +6,9 @@
   resource :user_sessions, only: [:create]
   root 'home#index'
   get  "first-edition" => "home#first_edition_projects"
-  get "sponsors" => "home#sponsors"
+  get  "sponsors" => "home#sponsors"
+  get  "mentee-profile" => "mentee_profile#index"
+  get  "mentee-dashboard" => "mentee_profile#dashboard"
```

Comment:
in rails it's recommended to use snake case so please change it to `mentee-dashboard`